### ISSUE TITLE
BigInt literals are broken.

### ISSUE THREAD

**Issue by alula** [javascript-obfuscator/javascript-obfuscator#588]
## Expected Behavior
BigInt literals are supported properly as normal numbers.

## Current Behavior
ESNext bigint literals are converted to normal numbers, resulting in broken code.

```
function a(b){return b<<10&4660;}a(123);
```

# Stack trace
```
TypeError: Cannot mix BigInt and other types, use explicit conversions
```

## Your Environment
* Obfuscator version used: 0.27.2
* Node version used: v12.10.0

## Minimal working example that will help to reproduce issue
```javascript
function test(a) {
  return a << 10n & 0x1234n
}
test(123n);
```

**Comment by sanex3339** [comment]
`escodegen` currently doesn't support it

**Comment by sanex3339** [comment]
Hmm. Looks like I can fix it on my side.

**Comment by sanex3339** [comment]
https://github.com/javascript-obfuscator/javascript-obfuscator/pull/599/files

**Comment by sanex3339** [comment]
Support in master. Expect a release later today.

**Comment by sanex3339** [comment]
Released as `0.28.0`
